src/pages/landing/onboarding/index.tsx

```diff
@@ -1,43 +1,198 @@
 import * as Yup from 'yup';
 import { Formik, Form, Field, ErrorMessage } from 'formik';
 import { useCreateBuilderPlaceMutation } from '../../../modules/BuilderPlace/hooks/UseCreateBuilderPlaceMutation';
-import { generateSubdomainPrefix } from '../../../modules/BuilderPlace/utils';
-import { useState } from 'react';
+import { showErrorTransactionToast } from '../../../utils/toast';
+import { PreferredWorkType } from '../../../types';
+import { useRouter } from 'next/router';
+import { upload } from '../../../modules/BuilderPlace/request';
+import * as fs from 'fs';
+import { generateSubdomainPrefix, slugify } from '../../../modules/BuilderPlace/utils';
 
+interface IFormValues {
+  name: string;
+  presentation: string;
+  preferred_work_type: PreferredWorkType[];
+  image_url: string;
+}
 function onboardingStep1() {
   const { data: createdBuilderPlace, mutateAsync: createBuilderPlaceAsync } =
     useCreateBuilderPlaceMutation();
-  const [name, setName] = useState('');
-
-  const sendDomain = async () => {
-    const subdomainPrefix = generateSubdomainPrefix(name);
-    const subdomain = `${subdomainPrefix}.${process.env.NEXT_PUBLIC_ROOT_DOMAIN}`;
-    await createBuilderPlaceAsync({
-      subdomain: subdomain,
-      name: name,
-      primaryColor: '#ffffff',
-      secondaryColor: '#ffffff',
-      ownerTalentLayerId: '1',
-    });
-    window.location.href = `${window.location.protocol}//${subdomain}/dashboard`;
+  const router = useRouter();
+
+  const initialValues: IFormValues = {
+    name: '',
+    presentation: '',
+    preferred_work_type: [PreferredWorkType.jobs],
+    image_url: '',
+  };
+
+  const validationSchema = Yup.object({
```

Comment:
did you have a problem here ?

Reply:
Just did this for testing upload, so I wouldn't have to refill all fields